### PR TITLE
Favourites/RemoveFavourite Servlets testing

### PR DESCRIPTION
**Pull request by azhara-a**
**Write FavouritesServletTest and RemoveFavouriteServletTest**

### CODE REVIEW

**Review by lminchillo — APPROVED** [review]
LGTM

File: backend/src/main/java/com/google/sps/data/Recipe.java [hunk 1]
```diff
@@ -65,6 +65,38 @@ public boolean containsIngredient(String ingredient) {
     return false;
   }
 
+  public String getName() {
```
Comment (mugdhalakhani) [inline backend/src/main/java/com/google/sps/data/Recipe.java:68]: nit:annotate

File: backend/src/test/java/com/google/sps/utils/TestUtils.java [hunk 2]
```diff
@@ -100,4 +100,41 @@ public static Entity getUserEntityWithAllergiesOnly() {
     user.setProperty(UserConstants.PROPERTY_ALLERGIES, ALLERGIES);
     return user;
   }
+
+  public static Recipe getRecipe() {
```
Comment (mugdhalakhani) [inline backend/src/test/java/com/google/sps/utils/TestUtils.java:104]: nit: Can we call this createTestRecipe() to clarify what the method is doing?

File: backend/src/test/java/com/google/sps/utils/TestUtils.java [hunk 3]
```diff
@@ -100,4 +100,41 @@ public static Entity getUserEntityWithAllergiesOnly() {
...
+        Arrays.asList(
+            "Tip the lemons, sugar and half the water "
+                + "into a food processor and blend until the lemon is finely chopped.",
+            "Pour the mixture into a sieve over a bowl, then press through as much "
+                + "juice as you can. Top up with the remaining water and serve with plain "
+                + "ice or frozen with slices of lemon and lime.");
+    return new Recipe(name, time, calories, difficulty, imageUrl, diet, ingredients, instructions);
+  }
+
+  public static Entity getRecipeEntity() {
```
Comment (mugdhalakhani) [inline backend/src/test/java/com/google/sps/utils/TestUtils.java:122]: nit: Can we call this createTestRecipeEntity() to clarify what the method is doing?

**Review by mugdhalakhani — APPROVED** [review]
LGTM, thanks!

**Pull request merged**